src/components/CR8 - GalleryText/GalleryText.vue

```diff
@@ -0,0 +1,234 @@
+<template>
+    <div class="gallery-text">
+        <!-- Title -->
+        <p class="title">Our Ecosystem</p>
+
+        <!-- Subtitle w/ Arrow -->
+        <p class="subtitle">
+            <span class="subtitle-text">We develop and produce original content for all genres and formats.</span>
+            <span class="prompt">
+                <p class="prompt-text">EXPLORE</p>
+                <span class="arrow">Arrow</span>
+            </span>
+        </p>
+
+        <!-- Block Links -->
+        <nuxt-link
+            v-for="(brand, index) in brands"
+            :key="index" 
+            class="brand-link"
+            @mouseover.native="set(brand.name)" 
+            @mouseout.native="set(null)"  
+            :to="`/${brand.name}`" tag="span"
+        >
+            <span class="logo">{{brand.name}} </span>{{brand.text}}
+        </nuxt-link>
+
+        <!-- Images absolute -->
+        <wp-image
+            v-for="(image, index) in images"
+            :key="`image-${index}`"
+            class="image"
+            :class="{'is-active': index == activeImage}"
+            :image="image"
+            mode="fullbleed"
+        />
```

Comment:
Currently the image will fade out, the show nothing for a split second, then show the next one. I think you'll want to wrap this in a `<transition>` component and tweak your code to `v-show` each image it should fix that. Or use the slideshow component I linked to in the review comment.

Also don't use the index as a key ever, you should use the `id` of the object ideally. Vue cache's these components using the key, so an index will never really work for that. If the same image with same `id` was used in a `wp-image` component somewhere else on the site, then it will be used form the cache and be faster!